app/views/layouts/application.html.erb

```diff
@@ -65,14 +65,15 @@
       <% end %>
 
       <% if flash["alert_with_description"] %>
+        <% alert = flash["alert_with_description"].stringify_keys %>
         <%= render "govuk_publishing_components/components/error_alert", {
-          message: flash["alert_with_description"].fetch("title"),
+          message: alert.fetch("title"),
           data_attributes: {
             gtm: "alert-with-description",
-            "gtm-value" => flash["alert_with_description"]["title"],
+            "gtm-value" => alert["title"],
```

Comment:
Why do we have two ways of retrieving the `"title"`?

Reply:
That'll be because this fetch is superfluous after the one on line 70. It does make for a jarring read though so I've amended it to be consistent.